validator/validation/readers.py

```diff
@@ -47,6 +48,10 @@ def create_reader(dataset, version):
 
     if dataset.short_name == globals.SMOS:
         reader = SMOSTs(folder_name, ioclass_kws={'read_bulk':True})
+        
+    if dataset.short_name == globals.ERA5:
```

Comment:
in globals ```ERA='ERA'```, which makes sense here as well, because the reader is the same across all ERA data products (there is not only "ERA5" but for example "ERA-Interim" which is the predecessor)